ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestsecurity.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+from functools import cached_property
+
+from ansible_collections.arista.avd.plugins.plugin_utils.eos_validate_state_utils.avdtestbase import AvdTestBase
+
+
+class AvdTestAPIHttpsSSL(AvdTestBase):
+    """
+    AvdTestAPIHttpsSSL class for verifies if eAPI HTTPS server SSL profile is configured and valid.
+    """
+
+    anta_module = "anta.tests.security"
+
+    @cached_property
+    def test_definition(self) -> dict:
```

Comment:
Given that we can return None (line 28)

```suggestion
    def test_definition(self) -> dict | None:
```